lib/generators/rspec.rb

```diff
@@ -20,6 +20,16 @@ def module_namespacing
           yield if block_given?
         end
       end
+
+      protected
+
+      def type_metatag(type)
+        if RUBY_VERSION > '1.8.7'
+          "type: :#{type}"
+        else
+          ":type => :#{type}"
+        end
+      end
```

Comment:
Thor throws an exception regarding arity if the method is public. Is it best to put this method in https://github.com/rspec/rspec-rails/blob/master/lib/rspec/rails/feature_check.rb?